### DISCUSSION TITLE
Remove the Gateway Terminology

### DISCUSSION THREAD

**Discussion by dlglin** [openwebwork/webwork2#1838] [Ideas]
The gateway quiz mode was originally written to handle gateway testing, but has evolved way beyond that, which means it's probably time to remove the "gateway" nomenclature, and just call them quizzes.

There are two aspects to this: mentions in the UI, and mentions on the back end.  The back end can be left alone as that is only seen by developers, but any strings displayed to users should probably move away from the gateway nomenclature.

**Comment by somiaj** [comment]
We could consider changing the name from 'quizzes'. The UI already does things like 'Grade Test', and 'Start New Test Version', so we could just call them tests.

**Comment by drgrice1** [comment]
I think removing the gateway nomenclature is a great idea.

I also agree with @somiaj  about considering changing them to tests.  The word "test" seems more generic to me.  A "quiz" is a short test.  An "exam" is a longer test.  But they are all tests.

**Comment by dlglin** [comment]
I'm okay with "test" rather than "quiz".  I mostly want to see the word gateway disappear.

**Comment by pstaabp** [comment]
I also agree with the removal of gateway and calling everything a test.

**Comment by dlglin** [comment]
Implemented in #1847
